platform/android/java/lib/src/org/godotengine/godot/input/GodotGestureHandler.kt

```diff
@@ -197,7 +197,10 @@ internal class GodotGestureHandler : SimpleOnGestureListener(), OnScaleGestureLi
 		if (event.actionMasked == MotionEvent.ACTION_UP) {
 			nextDownIsDoubleTap = false
 			GodotInputHandler.handleMotionEvent(event)
+		} else if (event.actionMasked == MotionEvent.ACTION_MOVE) {
```

Comment:
Thanks for the update, this looks like it should handle the issue!

There's one more scenario to consider and incorporate in the logic; when panning and scaling is enabled (`panningAndScalingEnabled == true`), the `ScaleGestureDetector` logic treats `double_tap` followed by `swipe` as a `magnify` event:
- For the documentation, see https://developer.android.com/reference/android/view/ScaleGestureDetector#setQuickScaleEnabled(boolean)
- And the scale callback is https://github.com/godotengine/godot/blob/cf8ad12b56df4ae7bba4c73070dd035693a880e4/platform/android/java/lib/src/org/godotengine/godot/input/GodotGestureHandler.kt#L242.

This capability is still private (not exposed to GDScript) and only being used by the Android editor to support scaling in the spatial editor.
To complement your fix and not break that functionality, you'll need to check that `panningAndScalingEnabled == false` before forwarding the motion event.